op-guide/docker.md

```diff
@@ -1,9 +1,196 @@
-# Docker
+# Run a TiDB Cluster in Docker on a Single Host
 
-## Run with `docker-compose`
+## Preparation
 
-A simple `docker-compose.yml`:
+Before you start, make sure that:
+
++ Installed the latest version of [Docker](https://www.docker.com/products/docker) 
++ Pulled the TiDB/TiKV/PD docker images from PingCAP's Docker Hub repositories
+`docker pull pingcap/tidb:latest`
+`docker pull pingcap/tikv:latest`
+`docker pull pingcap/pd:latest`
+
+## Create a docker bridge network
+
+```bash
+net="isolated_nw"
+docker network rm ${net}
+docker network create --driver bridge ${net} 
+```
+
+After creating a docker network, we will make all the TiDB containers added to it, which composed a standalone cluster.
```

Comment:
+After creating a Docker network, add all the TiDB containers to it to make a standalone cluster.